ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -88,6 +88,10 @@ wait "$!"
 
 sed -i '/^  channel:/d' "${dir}/manifests/cvo-overrides.yaml"
 
+if [[ ${OPENSHIFT_INSTALL_MASTERS_SCHEDULABLE} == "true" ]]; then
```

Comment:
As I recall the installer makes the master schedulable when there are no worker nodes. We should be able to drop this change.